### PR TITLE
Feat/info check failed

### PR DESCRIPTION
**Pull request by dmodalek**

### CODE REVIEW

File: docs/hooksInstalled.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Hooks Installed
+
+This warning means, that Git Hooks are not installed.
+
+In your `.git` folder there as is folder called `hooks`.
```
Comment (jegli) [inline docs/hooksInstalled.md:5]: ... there is a folder called ...

File: docs/lts.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+This warning means, that the Node Version which is installed on your machine is not an LTS Node Version. To change the Node version on your machine, consider using a Node Version Manager such as [NVS](https://github.com/jasongin/nvs).
+
+LTS stands for Long Term Support. LTS versions guarantee, that critical bugs will be fixed for a total of 30 months. Therefore it makes sense to use them in production application. 
+
+Even-numbered Node releases (10, 12, etc.) become Active LTS versions after six months of their initial release. From then on they are ready for general use. 
+
+To find the newest, active LTS version, have a look at the following table: [https://nodejs.org/en/about/releases](https://nodejs.org/en/about/releases). The one with the status `Active LTS` is the one you are looking for.
+
+Note: EnvLinter does not check, if the version is as ACTIVE LTS version. It only checks, if the version is an LTS version.
```
Comment (jegli) [inline docs/lts.md:11]: ... is **an** ACTIVE LTS ...

**Review by jegli — COMMENTED** [review]
Yay looks great @dmodalek ! 😍 I don't have a notebook at the moment to test it but from what I can see it looks amazing and will provide some very helpful information. 

I found two tiny typos which I commented.

I'm delighted to see that the env-linter is being further developed, thank you so much for your work! 😎✌️

Cheers from Spain ☀️

**Comment by dmodalek** [comment]
Hey @jegli did not expect feedback from you 😃 Thanks, I fixed the typos. 

Enjoy your travels 🌴

**Comment by danielkoch** [comment]
Cool PR. When do you plan to merge it?

**Comment by prothweil** [comment]
It is actually ready to merge, we just didn't have the time to do so. We should do that asap :)

**Pull request merged**